plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -149,7 +149,7 @@
          on to the appropriate Host (virtual host).
          Documentation at /docs/config/engine.html -->
 
-    <Engine name="Catalina" defaultHost="localhost">
+    <Engine name="Catalina" defaultHost="localhost" jvmRoute="${exo.cluster.node.name}">
```

Comment:
> What is the interest to put this variable here while the jvmRoute can be specified as a standard system property

Simplify configuration (IMO which is always interesting and the main purpose)

> How does the Tomcat Engine behave when the ${exo.cluster.node.name} var is empty

Nothing, it's the default value. Simply, nothing will be appended in Session ID

> This is not mandatory to realize load balancing with sticky session.

Yep, but more manual configuration stuff to care about

Our goal is to make the configuration of cluster as much simple as possible